### ISSUE TITLE
[PATCH product-offers] ConstraintViolationException.MissingRequiredParameters "Missing required parameters: ..."

### ISSUE THREAD

**Issue by vitalijr2** [allegro/allegro-api#8757]
Mamy ofertę **15341334213** i próbuję dodać do niej kolejny produkt. W tym celu używam PATCH z następującą zawartością:

```json
{
   "productSet":[
      {
         "product":{
            "id":"561d32eb-***-8b847b7424cc"
         }
      },
      {
         "product":{
            "id":"5435ceaf-***-85789189d977"
         }
      }
   ]
}
```

Allegro zwraca błąd:

```json
{
   "errors":[
      {
         "code":"ConstraintViolationException.MissingRequiredParameters",
         "message":"Missing required parameters: 209298, 356",
         "details":"ConstraintViolationException.MissingRequiredParameters",
         "path":"parameters",
         "userMessage":"Uzupełnij parametry obowiązkowe: Skład zestawu, Pojemność.",
         "metadata":{
            
         }
      }
   ]
}
```

Czy istnieje możliwość dodania produktu do oferty pomimo tego, że produkt główny nie posiada obowiązkowych parametrów?

**Trace-id:** 85f6af438acffda2

**Comment by MartaNowaczyk** [comment]
Nie, trzeba uzupełnić parametry obowiązkowe.

**Comment by vitalijr2** [comment]
Staram się filtrować takie oferty, aby zmniejszyć ilość niepotrzebnych próśb o aktualizację, korzystam w tym celu z zasobu `/sale/offers/unfilled-parameters`. 

Ten zasob zwraca mi 9 ofert.
**Trace-id:** 6772281e2c36b19b

Ale nie jest to pełna lista i niemal od razu po tym powtarza się błąd z aktualizacją znanego już produktu, o którym pisałem na początku.
![screenshot_2024-04-02_15-18-09_633271044](https://github.com/allegro/allegro-api/assets/368917/2068099b-3512-4388-87ed-0cbd805b9a6f)

**Comment by MartaNowaczyk** [comment]
Dziękuje za szczegółowe dane. Wrócę z informacją jak tylko zweryfikujemy ten przypadek.

**Comment by MartaNowaczyk** [comment]
Sprawdziliśmy i gdy oferta była aktywna zwracaliśmy te parametry we wspomnianym ednpoincie, teraz gdy oferta jest zakończona nie będzie widoczna na liście.

**Comment by vitalijr2** [comment]
produkt ma status LISTED:
```json
"publication": {
  "status": "LISTED"
},
```

Alie ta oferta może być modyfikowana?

**Comment by vitalijr2** [comment]
czy mogę za pomocą **PATCH** zmienić pole publikacji.status z `ACTIVE` z powrotem na `INACTIVE`?

**Comment by vitalijr2** [comment]
Czy dobrze rozumiem, że nie ma możliwości zmiany statusu **publication.status** z `ACTIVE` z powrotem na `INACTIVE`?

**Comment by MartaNowaczyk** [comment]
Tak, nie możesz zmienić statusu z "ACTIVE" na "INACTIVE", a jedynie na "ENDED".
